### PR TITLE
Dockerfile for generator

### PR DESCRIPTION
**Pull request by treeder**
Build with:

```sh
docker build -t asyncapi/generator .
```

Would be good to auto build and release this along with the npm releases.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# install dependencies
+COPY package*.json ./
+RUN npm install
+
+# copy code
+COPY . .
+# RUN chmod a+x /app/dockerboot.sh
+
+# CMD [ "node", "cli.js" ]
```
Comment (fmvilas) [inline Dockerfile]: Should we get rid of this commented line and the one above?

Comment (treeder) [reply Dockerfile]: Yes. :) Done.

**Comment by treeder** [comment]
Grabbed the `asyncapi` org on Docker Hub too, happy to hand it over to if you give me your Docker Hub username.

**Comment by fmvilas** [comment]
Awesome! My Docker Hub username is `fmvilas`. Very kind of you. Thanks!

**Pull request merged**